test-requirements.txt

```
@@ -0,0 +1 @@
+odoo-addon-account_payment_return@git+https://github.com/OCA/account-payment.git@refs/pull/792/head#subdirectory=account_payment_return
```

```suggestion
odoo-addon-account_payment_return@git+https://github.com/OCA/account-payment.git@refs/pull/833/head#subdirectory=account_payment_return
```
Hi @JasminSForgeFlow. It seems that this is the dependency (https://github.com/OCA/account-payment/pull/833) that's active now. The other PR is closed.

---

Updated, Thanks